auslib/web/common/history.py

```diff
@@ -53,6 +53,20 @@ def get_history(self, response_key='revisions'):
         return jsonify(ret)
 
 
+def get_input_dict():
+    reserved_filter_params = ['limit', 'page', 'timestamp_from', 'timestamp_to']
+    args = request.args
+    query_keys = []
+    query = {}
+    for key in args:
+        if key not in reserved_filter_params:
+            query_keys.append(key)
+
+    for key in query_keys:
+        query[key] = request.args.get(key)
+    return jsonify(query_keys=query_keys, query=query)
```

Comment:
`jsonify` should only be used when returning an HTTP Response - there's no point in returning JSON back to other Python functions -- just return a dict or tuple.

Reply:
Got that! I've fixed it now!